corehq/apps/analytics/tasks.py

```diff
@@ -277,7 +277,10 @@ def _send_hubspot_form_request(hubspot_id, form_id, data):
         form_id=form_id
     )
     response = requests.post(url, data=data)
-    metrics_counter('commcare.hubspot.sent_form', tags={'status_code': response.status_code})
+    metrics_counter('commcare.hubspot.sent_form', tags={
+        'status_code': response.status_code,
+        'form_id': form_id,
```

Comment:
Is this a high-cardinality tag? I think that is generally discouraged with datadog (possibly affects performance and/or cost).

Reply:
No. We have a very specific set of Hubspot Form IDs on HQ that are all referenced (or at least will be) in our Hubspot documentation. https://github.com/dimagi/commcare-hq/blob/a7923f5a5300feb423cd7ee22d780e5a4bcc8e87/corehq/apps/analytics/tasks.py#L60-L73